packages/matplotlib/test_matplotlib.py

```diff
@@ -1,12 +1,21 @@
+import pytest
+
+
+@pytest.mark.norefs
```

Comment:
```suggestion
@pytest.mark.skip_refcount_check
```

here and everywhere to be more explicit